### ISSUE TITLE
`dup_reports` missing from `general` endpoint in API

### ISSUE THREAD

**Issue by stucka** [GSA-TTS/FAC#2317]
There appears to have been some work done to implement `dup_reports` but it is not available in the `general` endpoint. It appears in the Census CSV's general file as `DUP_REPORTS` , where it is defined as "Whether or not the financial statements include departments that have separate expenditures not included in this audit"

**Comment by stucka** [comment]
I don't know if the `dup_reports` that has some work behind it is identical to `is_duplicate_reports` , discussed here as something with no new data:
https://github.com/GSA-TTS/FAC/issues/2249#issuecomment-1740020522

**Comment by jadudm** [comment]
Hi @stucka ,

I'm going to close this report after I make sure I've ticked the `dup` question fully (and I'll reference this ticket there). Your ticket came in at roughly the same time we were discussing this field with HHS, and I realized that:

1. Yes, we do collect this, and 
2. It was unclear to us what source we should use for `dup_reports`. 

We'll update the API shortly with this field. Thank you for the catch.
